ceiling-tests/v2/README.md

```diff
@@ -0,0 +1,606 @@
+| **2XL.440** | `m7i.4xlarge`  | 4 x `c7i.2xlarge` | 4 x `db.r7g.2xlarge` | No     | **51572**           | CPU            |
+| **2XL.550** | `m7i.4xlarge`  | 5 x `c7i.2xlarge` | 5 x `db.r7g.2xlarge` | No     | **61461**           | CPU            |
+| **2XL.552** | `m7i.4xlarge`  | 5 x `c7i.2xlarge` | 5 x `db.r7g.2xlarge` | Yes    | **58851**           | CPU (ES CPU)   |
+
+#### Family 4XL
+
+|             | **Proxy type** | **App nodes**     | **DB nodes**         | **ES** | **Supported users** | **Bottleneck**                 |
+|-------------|----------------|-------------------|----------------------|--------|---------------------|--------------------------------|
+| **4XL.440** | `m6in.8xlarge` | 4 x `c7i.4xlarge` | 4 x `db.r7g.4xlarge` | No     | **80573**           | CPU                            |
+| **4XL.442** | `m6in.8xlarge` | 4 x `c7i.4xlarge` | 4 x `db.r7g.4xlarge` | Yes    | **80532**           | CPU (ES CPU)                   |
+| **4XL.550** | `m6in.8xlarge` | 5 x `c7i.4xlarge` | 5 x `db.r7g.4xlarge` | No     | **93808**           | CPU                            |
+| **4XL.552** | `m6in.8xlarge` | 5 x `c7i.4xlarge` | 5 x `db.r7g.4xlarge` | Yes    | **91571**           | CPU (network, cluster, ES CPU) |
+| **4XL.662** | `m6in.8xlarge` | 6 x `c7i.4xlarge` | 6 x `db.r7g.4xlarge` | Yes    | **100926**          | CPU (network, cluster, ES CPU) |
+
+
+### Insights
+
+Apart from the raw numbers in the previous table, running these tests revealed some key points:
+
+- The main bottleneck we see across most of the tests is CPU.
+- Elasticsearch: Ceiling Tests v2 is the first time we add Elasticsearch to a batch of performance tests in a structured way. We learned some interesting things:
+  - Elasticsearch improved performance by an order of magnitude in the lower-end tests, when the main bottleneck was the DB. This proves that Elasticsearch is the best solution when the database is struggling.
+  - However, ES did absolutely nothing in the high-end tests, that were always bound by CPU.
+  - Hence, for Elasticsearch to improve stuff, the bottleneck *has to be* in the database.
+  - We used 2 nodes with a fixed instance type for all tests. The CPU of the instances maxed out starting with ~30k users. Thus, the potential improvement attributed to Elasticsearch is actually lost in tests with more than 30k users. We need to scale it, either vertically or horizontally, along with the expected number of supported users to see an improvement. We need more tests to provide a solid answer to this.
+- Network:
+  - High bandwidth is definitely a bottleneck in higher scales. This happens in micro-bursts, not in the average usage of the network, which comfortably sits below the thresholds set by the instance types used in the tests. However, due to usage peaks, we had to bump the originally planned proxy instance type for it to be able to handle the high load.
```

Comment:
> However, due to usage peaks, we had to bump the originally planned proxy instance type for it to be able to handle the high load.

Was this after the NIC rx ring buffer tunings?

Reply:
Do you mean the fix in https://github.com/mattermost/mattermost-load-test-ng/pull/778, right? It seems we bumped the instance after we fixed that. See 
![image](https://github.com/user-attachments/assets/459480d1-5ac4-4875-8f4e-896c5d60ba44)

Reply:
Ah, I think the core problem was we weren't using a network-optimized instance.